internal/config/view.go

```diff
@@ -0,0 +1,290 @@
+		// ELSE IF CONDITION:
+		// If there are arguments == 2 i.e. command looks like --> flow config view account <accountname>,
+		// Then we print all the details of the account "<accountname>".
+		// 	If the <accountname> doesn't exist in the configuration, then we print "Account <accountname> does not exist".
+
+		// Declaring a string variable to print our final list.
+		var list string = ""
+
+		if len(args) == 1 {
+			for _, value := range *state.Accounts() {
+				if count == 0 {
+					list = list + string(value.Name())
+					count = count + 1
+				} else if count > 0 {
+					list = list + ", " + string(value.Name())
+				}
+			}
+			// Return the list of accounts.
+			return &Result{
+				result: fmt.Sprintf("List of Accounts: %s", list),
+			}, nil
+		} else if len(args) == 2 {
+			for _, value := range state.Config().Accounts {
+				if value.Name == args[1] {
+					// Return the properties of a particular account.
+					return &Result{
+						result: fmt.Sprintf("Account Name: %s \nAddress: %s \nKey Properties: \nType: %s, Index: %d, Signature Algorithm: %s, Hash Algorithm: %s, Private Key: %s", value.Name, value.Address, value.Key.Type, value.Key.Index, value.Key.SigAlgo, value.Key.HashAlgo, value.Key.PrivateKey),
+					}, nil
+				}
+			}
+			return &Result{
+				result: fmt.Sprintf("Account %s does not exist", args[1]),
+			}, nil
+		}
+	}
+
+	// If the argument passed is "network".
+	if args[0] == "network" {
+
+		// Count variable for printing format.
+		var count int = 0
+
+		// IF CONDITION:
+		// If there is one argument in the command i.e. command looks like --> flow config view network,
+		// Then we print the list of all the present accounts in the configuration.
+		// ELSE IF CONDITION:
+		// If there are arguments == 2 i.e. command looks like --> flow config view network <networkname>,
+		// Then we print all the details of the network "<networkname>".
+		// 	If the <networkname> doesn't exist in the configuration, then we print "Network <networkname> does not exist".
+
+		// Declaring a string variable to print our final list.
+		var list string = ""
+
+		if len(args) == 1 {
+			for _, value := range *state.Networks() {
+				if count == 0 {
+					list = list + string(value.Name)
+					count = count + 1
+				} else if count > 0 {
+					list = list + ", " + string(value.Name)
+				}
+			}
+			// Return the list of networks.
+			return &Result{
+				result: fmt.Sprintf("List of Networks: %s", list),
+			}, nil
+		} else if len(args) == 2 {
+			for _, value := range state.Config().Networks {
+				if value.Name == args[1] {
+					// Return the properties of a particular network.
+					return &Result{
+						result: fmt.Sprintf("Network Name: %s \nHost: %s \n", value.Name, value.Host),
+					}, nil
+				}
+			}
+			return &Result{
+				result: fmt.Sprintf("Network %s does not exist", args[1]),
+			}, nil
+		}
+	}
+
+	// If the argument passed is "contract".
+	if args[0] == "contract" {
```

Comment:
It seems that each of these blocks does the same thing with different values which is a good idea to extract into a function.